pkgs/development/libraries/glog/default.nix

```diff
@@ -21,22 +21,24 @@ stdenv.mkDerivation rec {
     "-DBUILD_SHARED_LIBS=ON"
   ];
 
-  # TODO: Re-enable Darwin tests once we're on a release that has https://github.com/google/glog/issues/709#issuecomment-960381653 fixed
-  doCheck = !stdenv.isDarwin;
```

Comment:
Needs `doCheck = true;`. Removing this line has disabled tests on all platforms

Reply:
I don't know why but for some reason I thought this was on by default now.